spec/Expressions.tex

```diff
@@ -1361,30 +1361,29 @@ \subsection{The String Concatenation Operator}
 \index{operators!concatenation!string}
 \index{operators!+ (string)@\chpl{+} (string)}
 
-The string concatenation operator \chpl{+} is predefined over numeric, boolean,
-and enumerated types with strings. It casts its operands to string type and
-concatenates them together.
+The string concatenation operator \chpl{+} is predefined for string
+arguments and returns a new string that is the concatenation of its
+arguments:
+
+\begin{chapel}
+proc +(s0: string, s1: string): string
+\end{chapel}
 
 \begin{chapelexample}{string-concat.chpl}
-The code
-\begin{chapelnoprint}
-var i:int = 3;
-writeln(
-\end{chapelnoprint}
+The code:
 \begin{chapel}
-"result: "+i
+var x: string = "hi";
+var y: string = " there";
+var z = x + y;
 \end{chapel}
 \begin{chapelnoprint}
-);
+writeln(z);
 \end{chapelnoprint}
 \begin{chapeloutput}
-result: 3
+hi there
 \end{chapeloutput}
-where \chpl{i} is an integer appends the string representation of \chpl{i} to the
-string literal \chpl{"result: "}.  If \chpl{i} is \chpl{3}, then the resulting string
-would be \chpl{"result: 3"}.
-\begin{chapelnoprint}
-\end{chapelnoprint}
+will cause \chpl{z} to be a new string containing the value \chpl"hi
```

Comment:
Should that be `\chpl{"hi` at the end of the line?

Reply:
Sheesh, sloppy...  thanks for the catch!